src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -47,93 +70,1248 @@ static ColliderCylinderInit D_80A9A100 = {
     },
     { 20, 46, 0, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ko/func_80A96DB0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ko/func_80A96EC4.s")
+CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
+
+static void* sFa_eyes[] = { gFaEyeOpenTex, gFaEyeHalfTex, gFaEyeClosedTex, NULL };
+static void* sKw1_eyes[] = { 0x06000F4C, 0x06001A0C, 0x06001E0C, NULL };
+
+typedef struct {
+    s16 objectId;
+    Gfx* dList;
+    void** eyeTextures;
+} EnKoHead;
+
+EnKoHead sHead[] = { { OBJECT_KM1, 0x06001890, NULL },
+                     { OBJECT_KW1, 0x06002C10, sKw1_eyes },
+                     { OBJECT_FA, gFaDL, sFa_eyes } };
+
+typedef struct {
+    s16 objectId;
+    FlexSkeletonHeader* flexSkeletonHeader;
+} EnKoSkeleton;
+
+static EnKoSkeleton sSkeleton[2] = { { OBJECT_KM1, 0x060000F0 }, { OBJECT_KW1, 0x060000F0 } };
```

Comment:
```suggestion
static EnKoSkeleton sSkeleton[2] = { { OBJECT_KM1, gKm1Skel }, { OBJECT_KW1, gKw1Skel } };
```

Reply:
Km1 and Kw1 are non-matching due to ZAPD issue